AzerothAuctionAssassin.py

```
@@ -3565,6 +3596,185 @@ def isfloat(self, value):
+                    # If we have specific item IDs, create an entry for each one
+                    for item_id in rule["item_ids"]:
+                        item_match = self.item_statistics[
+                            self.item_statistics["itemID"] == item_id
+                        ]
+                        item_name = ""
+                        if not item_match.empty:
+                            item_name = f'"{item_match.iloc[0]["itemName"]}"'
+
+                        # Construct PBS entry for this specific item
+                        pbs_entry = (
+                            f"Snipe?"
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)